backend/development.ini.sample

```diff
@@ -101,6 +101,14 @@ basic_setup.session_secret = change_this_value_please!
 # be loaded, allowing to add new feature to an existing Tracim.
 ; plugin.folder_path = %(here)s/plugins
 
+### Workspace ###
+# list of access_type allowed for new workspace. Access_type available in Tracim are:
+# - confidential: not visible publicly, access only by invitation workspace_manager/admin
+# - on_request: publicly visible, access by invitation or request with validation by workspace manager/admin)
+# - open: publicly visible and access by invitation or request without validation
+; workspace.allowed_access_types = confidential,on_request,open
```

Comment:
Maybe add this is the default value?

Reply:
default value are all showed this way with the ";" it's explain at the beginning of the config file.

Reply:
Config file begin with:
```ini
## File Syntax
# Comments start by `;` or `#`.
# In the sample config file:
#  - `;` commented lines are used to indicate the default values used by Tracim;
#  - `#` commented lines are used for comments/examples.
# Boolean values can be `True` or `False`
# All parameters starting with `basic_setup` can be used as variables in other parameters.
# For simple needs they are all you need to customize.

```
With this documentation, Is it ok for you ?

Reply:
Mostly, I just now just find strange to put the default commented value before the example, I would have put the example before.

Reply:
I don't have much opinion on this subject, we do in this order at multiple place in config file without any trouble until now.
I suggest to get opinion from both @PhilippeAccorsi and @raphj and change the order according to that if needed in another issue .

Reply:
Yes, this is also why I approved the changes: my comment is not blocking.

Reply:
I agree with @grignards, we should put the example before the default value.

Reply:
If I remember correctly we have done a big modification in development.ini file in Tracim 2.7.x with new naming process and validate this choice (with some others dev). Modification in this file is not simple on existing installation so for me its important to define structure of file for a long time period.

I think its not necessary to change structure of this file each years ;)

So I think we can create an issue about wording of this file, and try to find what is the better solution (what is done on other projet, what we really want, etc.)